### PR TITLE
MTP-1537: Move dependencies in mtp-common

### PR DESCRIPTION
**Pull request by xoen**
This is using a newer version of `mtp-common` (`11.1.0`).

⚠️ **NOTE**: as we bump from `10.x` to `11.x` and that new version contains some other changes (e.g. GDS design system changes) - this probably need some coordination before we merge it.

Ticket: https://dsdmoj.atlassian.net/browse/MTP-1537

### CODE REVIEW

**Comment by xoen** [comment]
FYI: Rebased, I think it can be merged as soon as tests pass again after rebase and someone approves this.

**Review by ushkarev — APPROVED** [review]

**Comment by xoen** [comment]
Thanks @ushkarev ☺️

**Pull request merged**
